apps/mobile/src/util/navigation.js

```diff
@@ -213,10 +212,6 @@ export function removeGroupFromUrl (url) {
   return url.replace(new RegExp(matchForReplaceRegex), '')
 }
 
-export function gotoExternalUrl (url) {
```

Comment:
This is still used in MemberProfile. It should probably be replaced by openURL there, but if you don't want to convert and test that area then just leave this for now.

Reply:
Strike that, I was wrong. Sorry. Saw it in Web and thought I was seeing a Mobile search result. Yes, removing this is great :)